.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug report
+about: Create a report to help us improve Racket 
+title: ''
+labels: bug
```

Comment:
I don't think this needs a label by default.

Reply:
I kinda disagree. Label makes it possible to filter issues. I think we should keep it.

Reply:
Labels are great, but I'm not sure "bug" is a useful one.

Reply:
IIUC, it's to distinguish bugs from feature requests.

Granted, Racket seems to have bug issues far more than feature request issues.

Reply:
I've removed the label - everything is a bug if it is not a feature request. Note the current cohort of bug reports don't have a label so this is consistent.